src/player/player.py

```diff
@@ -44,6 +54,18 @@ def move(self, move):
 
     def set_xy(self, xy):
         self.rect.center = (xy[X], xy[Y])
+        self.x = xy[X]
+        self.y = xy[Y]
+
+    def player_animation_controller(self, move, local_window):
+        frame = move % FRAMES_TO_ANIMATE
+        if frame < FRAMES_BOUNDARY_TO_START_ANIMATE or frame > FRAMES_BOUNDARY_TO_STOP_ANIMATE:
+            local_window.screen.blit(self.skin, self.rect)
+        elif FRAMES_BOUNDARY_TO_START_ANIMATE <= frame <= FRAMES_BOUNDARY_TO_STOP_ANIMATE:
+            local_window.screen.blit(self.skin_animation, self.rect)
+
 
     def print_player(self, local_window):
-        local_window.screen.blit(self.skin, self.rect)
+        self.player_animation_controller(self.pos, local_window)
+        self.pos += 1
```

Comment:
I think we should try make this variable 0 after time if we can. I'm afraid it would be too big after time